### ISSUE TITLE
Stuck in fullscreen

### ISSUE THREAD

**Issue by Quixote1** [milkdrop2077/MilkDrop3#44]
I'll preface this post by praising your work and thanking you for refreshing this beautiful masterpiece. Truly appreciative of your work!

Now for the issue at hand, I found myself stuck in fullscreen on my first run. From what I gathered, I'm supposed to be able to hit escape and leave FS, but I had to mess around with task manager, etc. in order to maintain control of my PC without a reboot. I have yet to retest before reporting in case I get stuck again and give up momentarily. (already troubleshooting a network problem that's got me ready to call it a day).
I'm on a fully updated Windows 10 system. Let me know if you need any other info or need me to test anything.
Thanks!  I'll go play around with it some more and see if I experience a similar lockup.

**Comment by Quixote1** [comment]
Sorry, I think that may have been a brain-fart. It's Alt-Enter, right?  :/   Sorry.   Escape would be a nice key though. Consider it a request, I guess.

**Comment by milkdrop2077** [comment]
yes Alt-enter or double click with your mousse! I'll consider the escape key

**Comment by Quixote1** [comment]
Thanks, I missed that initially and went for the escape key for some reason. Sorry, and thanks for bringing this to us. You're an amazing person.

**Comment by Grooves1210** [comment]
Escape key is already assigned a function (for editing presets)
